Doc/library/datetime.rst

```diff
@@ -752,6 +752,13 @@ Other constructors, all class methods:
    :class:`.datetime` object.  An aware current UTC datetime can be obtained by
    calling ``datetime.now(timezone.utc)``.  See also :meth:`now`.
 
+   .. warning::
+
+      Because naive datetimes are treated by many functions as local times,
```

Comment:
This should be a bit more specific. Also, code comments or Sphinx roles should be used when referencing objects (such as classes, methods, or functions). In this case, code comments would be more appropriate since this is already within the section for ``datetime``.

```suggestion
      Because naive ``datetime`` objects are treated by many ``datetime`` methods as local times,
```